students/Wieslaw_Pucilowski/lesson02/FizzBuzz.py

```diff
@@ -0,0 +1,19 @@
+__author__ = 'Wieslaw Pucilowski'
+
+def FizzBuzz():
```

Comment:
Per PEP 8 function names should be lower case with words separated by underscores.  i.e., fizz_buzz():